amd64.env

```diff
@@ -2,6 +2,7 @@ rosdistro=galactic
 rmw_implementation=rmw_cyclonedds_cpp
 base_image=ubuntu:20.04
 cuda_base_image=nvidia/cuda:11.6.2-devel-ubuntu20.04
+cuda_runtime_base_image=nvidia/cuda:11.6.2-runtime-ubuntu20.04
```

Comment:
Do you want to use this image for both `cuda` and `no-cuda` images?
If not, you need to switch the base image like this.
https://github.com/autowarefoundation/autoware/blob/b64cfdb55c2106934b5262abd8667c65671ed597/.github/workflows/docker-build-and-push-main.yaml#L21-L28

Reply:
If CUDA runtime is light, I think it's okay to use the common base image for both images.

Reply:
Yes, agree, I will directly change cuda_base_image to 11.6.2-runtime-ubuntu20.04